src/components/card.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */

```

Comment:
- [ ] @Sammy3000 it's not good practice to disable the linter check in your file, kindly fix the linter errors instead of disabling them.